src/Compilers/Core/Portable/PEWriter/MetadataWriter.cs

```diff
@@ -457,6 +457,8 @@ private bool IsMinimalDelta
         protected readonly List<MethodImplementation> methodImplList = new List<MethodImplementation>();
         private readonly Dictionary<IGenericMethodInstanceReference, BlobHandle> _methodInstanceSignatureIndex = new Dictionary<IGenericMethodInstanceReference, BlobHandle>(ReferenceEqualityComparer.Instance);
 
+        protected readonly List<EntityHandle> _customAttributeTargets = new List<EntityHandle>();
+
```

Comment:
I think we should move this to the DeltaMetadataWriter subclass since it's not needed unless we are emitting EnC delta.